tests/test_cli_data.py

```diff
@@ -67,78 +80,65 @@ def test_data(config):
     assert "List data in managed data on Deep Origin" in stdout, "Unexpected output"
 
 
-def test_describe_file(config):
+@pytest.mark.parametrize("json_option", JSON_OPTIONS)
```

Comment:
note this allows us to test every CLI command with both the normal (human) output, and `--json` 

this happens automatically -- pytest handles this for us